### PR TITLE
added rego rules for ec2 and iam for AWS cloud

### PR DESCRIPTION
**Pull request by jaiminswan**
EC2:
- PR-AWS-CLD-EC2-006

IAM:
- PR-AWS-CLD-IAM-011
- PR-AWS-CLD-IAM-012
- PR-AWS-CLD-IAM-013
- PR-AWS-CLD-IAM-014
- PR-AWS-CLD-IAM-015
- PR-AWS-CLD-IAM-016
- PR-AWS-CLD-IAM-017
- PR-AWS-CLD-IAM-018
- PR-AWS-CLD-IAM-019
- PR-AWS-CLD-IAM-020
- PR-AWS-CLD-IAM-021
- PR-AWS-CLD-IAM-022
- PR-AWS-CLD-IAM-023
- PR-AWS-CLD-IAM-024
- PR-AWS-CLD-IAM-025
- PR-AWS-CLD-IAM-026
- PR-AWS-CLD-IAM-027

API_Gateway:
- PR-AWS-CLD-AG-008

App_Sync
- PR-AWS-CLD-APS-001

IAM:
- PR-AWS-TRF-IAM-011
- PR-AWS-TRF-IAM-012
- PR-AWS-TRF-IAM-013
- PR-AWS-TRF-IAM-014
- PR-AWS-TRF-IAM-015
- PR-AWS-TRF-IAM-016
- PR-AWS-TRF-IAM-017
- PR-AWS-TRF-IAM-018
- PR-AWS-TRF-IAM-019
- PR-AWS-TRF-IAM-020
- PR-AWS-TRF-IAM-021
- PR-AWS-TRF-IAM-022
- PR-AWS-TRF-IAM-023
- PR-AWS-TRF-IAM-024
- PR-AWS-TRF-IAM-025
- PR-AWS-TRF-IAM-026
- PR-AWS-TRF-IAM-027

API_Gateway:
- PR-AWS-TRF-AG-008

App_Sync
- PR-AWS-TRF-APS-001

IAM:
- PR-AWS-CFR-IAM-011
- PR-AWS-CFR-IAM-012
- PR-AWS-CFR-IAM-013
- PR-AWS-CFR-IAM-014
- PR-AWS-CFR-IAM-015
- PR-AWS-CFR-IAM-016
- PR-AWS-CFR-IAM-017
- PR-AWS-CFR-IAM-018
- PR-AWS-CFR-IAM-019
- PR-AWS-CFR-IAM-020
- PR-AWS-CFR-IAM-021
- PR-AWS-CFR-IAM-022
- PR-AWS-CFR-IAM-023
- PR-AWS-CFR-IAM-024
- PR-AWS-CFR-IAM-025
- PR-AWS-CFR-IAM-026
- PR-AWS-CFR-IAM-027

API_Gateway:
- PR-AWS-CFR-AG-008

App_Sync
- PR-AWS-CFR-APS-001

### CODE REVIEW

**Comment by farchide** [comment]
@jaiminswan we need to submit the updated master compliance test with the polciies
it should include the updated compliance, severity and the full description

**Review by farchide — COMMENTED** [review]
Please include the updated master compliance

File: aws/cloud/master-compliance-test.json [hunk 1]
```diff
@@ -526,6 +526,37 @@
...
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "AWS API Gateway REST API is not configured with AWS Web Application Firewall v2 (AWS WAFv2)",
+                    "description": "AWS API Gateway REST API which is not configured with AWS Web Application Firewall. As a best practice, enable the AWS WAF service on API Gateway REST API to protect against application layer attacks. To block malicious requests to your API Gateway REST API, define the block criteria in the WAF web access control list (web ACL).",
+                    "tags": [
+                        {
+                            "cloud": "AWS",
+                            "compliance": [
+                                "NIST 800"
```
Comment (farchide) [inline aws/cloud/master-compliance-test.json:552]: @jaiminswan we need full compliance list on each test case.

**Review by harshilswan — APPROVED** [review]

File: aws/cloud/master-compliance-test.json (hunk 1 above)
Comment (jaiminswan) [reply aws/cloud/master-compliance-test.json]: added more compliance

**Pull request merged**